src/services/buildService.ts

```diff
@@ -24,6 +24,16 @@ class BuildService {
   public getBuildCount(requestConfig: AxiosRequestConfig = {}) {
     return pncClient.getHttpClient().get(`${this.path}/count`, requestConfig);
   }
+
+  /**
+   * Gets dependency graph for a build.
+   * @returns DependencyGraph
```

Comment:
what does it return if id is falsy value?

Reply:
I removed the restriction to null id because it was not removed from other services like project service. also either a invalid format of id or id that has no data will be handled by data container.
![image](https://user-images.githubusercontent.com/6870455/185396656-3307fbee-c243-4e2c-a136-7750101b1eb8.png)
![image](https://user-images.githubusercontent.com/6870455/185396761-81890923-37aa-4fab-8475-8e85a8dea5db.png)